pkg/options/options.go

```diff
@@ -64,3 +67,27 @@ func (o *Options) Summarize(logger *logrus.Entry) {
 func (o *Options) Logger() *logrus.Entry {
 	return o.Log.WithFields(logrus.Fields{"arch": o.Arch.ToAPK()})
 }
+
+// Tempdir returns the temporary directory where apko will create
+// the layer blobs
+func (o *Options) TempDir() string {
+	if o.TempDirPath != "" {
+		return o.TempDirPath
+	}
+
+	path, err := os.MkdirTemp(os.TempDir(), "apko-temp-*")
+	if err != nil {
+		o.Logger().Fatalf(fmt.Errorf("creating tempdir: %w", err).Error())
+	}
+	o.TempDirPath = path
+	return o.TempDirPath
+}
+
+// TarballFileName returns a deterministic filename for the layer taball
+func (o Options) TarballFileName() string {
+	tarName := "apko.tar.gz"
+	if o.Arch.String() != "" {
+		tarName = fmt.Sprintf("apko-%s.tar.gz", o.Arch.String())
```

Comment:
```suggestion
		tarName = fmt.Sprintf("apko-%s.tar.gz", o.Arch.ToAPK())
```

Otherwise we get filenames like `apko-arm/v6.tar.gz`.

Reply:
Looks like there are still some fallout here.  Digging into it.